[2024-03-04T10:00:19.127+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T09:58:02.790685+00:00 [queued]>
[2024-03-04T10:00:19.143+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T09:58:02.790685+00:00 [queued]>
[2024-03-04T10:00:19.144+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-03-04T10:00:19.165+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_load_weather_data_to_S3> on 2024-03-04 09:58:02.790685+00:00
[2024-03-04T10:00:19.176+0000] {standard_task_runner.py:60} INFO - Started process 129 to run task
[2024-03-04T10:00:19.180+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data_to_S3', 'manual__2024-03-04T09:58:02.790685+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmp52pzs6a8']
[2024-03-04T10:00:19.182+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask transform_load_weather_data_to_S3
[2024-03-04T10:00:19.263+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T09:58:02.790685+00:00 [running]> on host 4f01ef8a3140
[2024-03-04T10:00:19.381+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data_to_S3' AIRFLOW_CTX_EXECUTION_DATE='2024-03-04T09:58:02.790685+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-04T09:58:02.790685+00:00'
[2024-03-04T10:00:20.226+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-04T10:00:20.240+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data_to_S3, execution_date=20240304T095802, start_date=20240304T100019, end_date=20240304T100020
[2024-03-04T10:00:20.281+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-04T10:00:20.318+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
